librz/core/linux_heap_glibc.c

```diff
@@ -1307,30 +1307,25 @@ static void GH(print_heap_bin)(RzCore *core, GHT m_arena, MallocState *main_aren
 		offset = 12 * SZ + sizeof(int) * 2;
 	}
 
-	switch (input[0]) {
-	case '\0': // dmhb
+	if (!input[0]) {
 		PRINT_YA("Bins {\n");
-		for (i = 0; i < NBINS - 1; i++) {
+		for (int i = 0; i < NBINS - 1; i++) {
```

Comment:
i hope `NBINS` is a macro, otherwise is a terrible idea to have `signed int` here.

Reply:
It is `NBINS = 128`. Changed it to `size_t` nonetheless.